flang/test/Driver/linker-flags.f90

```diff
@@ -2,8 +2,9 @@
 ! invocation. These libraries are added on top of other standard runtime
 ! libraries that the Clang driver will include.
 
-! RUN: %flang -### -target ppc64le-linux-gnu %S/Inputs/hello.f90 2>&1 | FileCheck %s --check-prefixes=CHECK,GNU
+! RUN: %flang -### -target ppc64le-linux-gnu %S/Inputs/hello.f90 2>&1 | FileCheck %s --check-prefixes=CHECK,UNIX
```

Comment:
while changing the lines, change deprecated `-target ` to `--target=`

Reply:
Done: since I'd already touched almost half the instances, I changed them all for consistency.